tests/version_test.go

```diff
@@ -10,7 +10,6 @@ var _ = Describe("Version", func() {
 	It("prints its version", func() {
 		output, err := execute("deis --version")
 		Expect(err).NotTo(HaveOccurred())
-		// TODO: read the expected version from ../client/deis-version?
-		Expect(output).To(Equal("2.0.0-dev\n"))
+		Expect(output).To(MatchRegexp("2.0.0-dev-[0-9a-f]{7}\n"))
```

Comment:
Also the dots indicate any character. Here's a regex based on @sgoings suggestion:

```
^\d\.\d\.\d(-dev-\w{7})?\n$
```